test/xds/xds_client_custom_lb_test.go

```diff
@@ -194,12 +201,31 @@ func (s) TestWrrLocality(t *testing.T) {
 				{addr: backend5.Address, count: 8},
 			},
 		},
+		{
+			name: "custom_lb_least_request",
+			wrrLocalityConfiguration: wrrLocality(&v3leastrequestpb.LeastRequest{
+				ChoiceCount: wrapperspb.UInt32(2),
+			}),
+			// Least request's randomness of indexes to sample (unary RPCs which
+			// don't persist so never any actual RPC counts over iterations)
```

Comment:
Something is incomplete in this sentence:

`unary RPCs which don't persist so never any actual RPC counts over iterations`

Reply:
Changed to "The test performs a Unary RPC, and blocks until the RPC returns,  and then makes the next RPC. Thus, over iterations, no RPC counts are present. This causes Least request's randomness of indexes to sample to converge onto a round robin distribution per locality. Thus, expect the same distribution as round robin above."